cli/root.go

```diff
@@ -121,7 +121,7 @@ func Root(subcommands []*cobra.Command) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:           "coder",
 		SilenceErrors: true,
```

Comment:
Should `SilenceErrors` be also set to `false` whilst we are in here?

Reply:
https://sourcegraph.com/github.com/spf13/cobra/-/blob/command.go?L226:2&subtree=true#tab=references

Reply:
**TODO:** Remove `Run 'coder login --help' for usage.`